src/dist/replication/lib/duplication/load_from_private_log.cpp

```diff
@@ -131,6 +151,25 @@ void load_from_private_log::replay_log_block()
                 _current->path(),
                 _start_offset);
             _counter_dup_load_file_failed_count->increment();
+            _err_file_repeats_num++;
+            if (will_fail_skip()) {
+                // skip this file
+                derror_replica("failed loading for {} times, abandon file {} and try next",
+                               _err_file_repeats_num,
+                               _current->path());
+                _err_file_repeats_num = 0;
+
+                auto prev_offset = _current_global_end_offset;
+                if (switch_to_next_log_file()) {
+                    // successfully skip to next file
+                    auto skipped_bytes = _current_global_end_offset - prev_offset;
+                    _counter_dup_load_skipped_bytes_count->add(skipped_bytes);
+                    repeat(_repeat_delay);
+                    return;
+                }
+            } else if (will_fail_fast()) {
```

Comment:
Could you add some suggestion in this log, then the cluster administrator know how to deal with this issue directly.

Reply:
Done. But I still can't provide much help info. In this case, the admin should check the coredump for the cause for loading failure.